src/Dashboard/Module/DashboardModule.php

```diff
@@ -0,0 +1,135 @@
+<?php
+namespace CrudView\Dashboard\Module;
+
+use Cake\View\Exception\MissingTemplateException;
+use Cake\View\ViewVarsTrait;
+use CrudView\Dashboard\Traits\DashboardTrait;
+use Error;
+use Exception;
+use ReflectionMethod;
+
+abstract class DashboardModule
```

Comment:
So `DashboardModule` is basically a stripped down version of `Cell`?

Reply:
Sort of. I don't really have time to refactor to use Cell as a class DashboardModule extends, but basically I want to be able to use both the Entity setters/getters (with some validation) and the Cell rendering in a single class.

Reply:
Sounds a bit like reinventing the wheel to me. Also I noticed that DashboardModule doesn't get access to request instance. It very likely some might want to do things based on request params or session.